.github/workflows/native.yml

```diff
@@ -47,3 +47,43 @@ jobs:
         working-directory: ./native
       - run: npm run test
         working-directory: ./native
+
+  lint:
+    name: Lint
+    needs: changes
+    if: ${{ needs.changes.outputs.native == 'true' }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 7
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: npm
+          cache-dependency-path: native/package-lock.json
+      - run: npm install -g npm
+      - run: npm install
```

Comment:
Just curious, why do we need the `npm install -g npm` here? `setup-node` includes a consistent version of npm, so we should be ok to just use that, right?

I also typically opt for `npm ci` instead of `npm install` in github actions. Helps make sure the package-lock is respected and up to date.

Reply:
I was basing it on the other yml files within the project, but I can try it with just `npm ci` and see how it goes